packages/verification-vue-component/src/components/Verification/items/cards/VerificationCard.vue

```diff
@@ -80,7 +80,8 @@
                     <div class="verifier-name">
                         <span class="label">{{ _$t('verification.result.meta.issuerVerifiedBy') }}</span>
                         <div v-if="verificationItem.issuerVerifiedImg" class="verifier-image">
-                            <img :src="verificationItem.issuerVerifiedImg" alt=""/>
+                            {{ /* Workaround because old verification tool should still use the old switch logo but the redesign should use a new switch logo, needs to be removed when event structure is final */ }}
```

Comment:
You can use a normal HTML comment `<!-- comment -->` this should not be visible in the production build.